Podfile.lock

```diff
@@ -60,7 +60,7 @@ PODS:
     - AppAuth/Core (~> 1.4)
     - GTMSessionFetcher/Core (~> 1.5)
   - GTMSessionFetcher/Core (1.7.0)
-  - Gutenberg (1.71.2):
+  - Gutenberg (1.71.1):
```

Comment:
This version should be `1.71.3` (looks like it was set as `1.71.2` because when the tag was created the version was `.2` ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/8a477bd2601e7abbf3bc1b64784aff07d4944dee/package.json#L3))). Therefore, it would be great if we create a new tag `v1.72.0-alpha4` to reflect this.

Reply:
In order to address this, I created a new tag in Gutenberg Mobile and updated its reference to `v1.72.0-alpha4`